ui/src/components/bedset-splash-components/header.tsx

```diff
@@ -104,16 +104,22 @@ export const BedsetSplashHeader = (props: Props) => {
         </div>
       </div>
       <div>
-        <p className="mb-2">{metadata?.description || 'No description available'}</p>
+        <p className='text-body-secondary fst-italic'>{metadata?.description || 'No description available'}</p>
       </div>
-      <div className="d-flex flex-row align-items-end justify-content-start gap-1">
-        <div className="badge bg-primary text-wrap">
-          <i className="bi bi-hash me-1" />
-          {metadata.md5sum}
-        </div>
-        <div className="badge bg-primary text-wrap">
-          <i className="bi bi-file-earmark-text me-1" />
-          {metadata.bed_ids?.length} BED files
+      <div className="d-flex flex-row align-items-end justify-content-between mt-2">
+        <div className="d-flex flex-row gap-1">
+          <p className='mb-0'>
+            <div className="badge bg-primary text-wrap">
+              <i className="bi bi-hash me-1" />
+              {metadata.md5sum}
+            </div>
+          </p>
+          <p className='mb-0'>
+            <div className="badge bg-primary text-wrap">
+              <i className="bi bi-file-earmark-text me-1" />
+              {metadata.bed_ids?.length} BED files
```

Comment:
Should we check for `bed_ids` first? Otherwise this will render as just ` BED files`. Small thing, just curious